src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/instance_hyjal.cpp

```diff
@@ -267,147 +267,148 @@ class instance_hyjal : public InstanceMapScript
+                    for (ObjectGuid const& guid : _baseAlliance)
                     {
-                        for (ObjectGuid const& guid : _baseAlliance)
+                        if (Creature* creature = instance->GetCreature(guid))
                         {
-                            if (Creature* creature = instance->GetCreature(guid))
+                            if (creature->IsAlive())
                             {
-                                if (creature->IsAlive())
-                                {
-                                    float x, y, z;
-                                    jaina->GetNearPoint(creature, x, y, z, 10.f, 0, jaina->GetAngle(creature));
-                                    creature->SetWalk(true);
-                                    creature->GetMotionMaster()->MovePoint(1, x, y, z);
-                                }
+                                float x, y, z;
+                                jaina->GetNearPoint(creature, x, y, z, 10.f, 0, jaina->GetAngle(creature));
+                                creature->SetWalk(true);
+                                creature->GetMotionMaster()->MovePoint(1, x, y, z);
                             }
                         }
                     }
+                }
+
+                // Despawn all alliance NPCs
+                scheduler.Schedule(21000ms, [this](TaskContext)
+                    {
+                        for (ObjectGuid const& guid : _baseAlliance)
+                        if (Creature* creature = instance->GetCreature(guid))
+                            creature->DespawnOrUnsummon();
 
-                    // Despawn all alliance NPCs
-                    scheduler.Schedule(21000ms, [this](TaskContext)
+                // Spawn Roaring Flame after a delay
+                scheduler.Schedule(30s, [this](TaskContext)
+                    {
+                        for (ObjectGuid const& guid : _roaringFlameAlliance)
                         {
-                            for (ObjectGuid const& guid : _baseAlliance)
-                                if (Creature* creature = instance->GetCreature(guid))
-                                    creature->DespawnOrUnsummon();
-
-                            // Spawn Roaring Flame after a delay
-                            scheduler.Schedule(30s, [this](TaskContext)
-                                {
-                                    for (ObjectGuid const& guid : _roaringFlameAlliance)
-                                    {
-                                        if (GameObject* flame = instance->GetGameObject(guid))
-                                            flame->Respawn();
-                                    }
-                                });
-                        });
+                            if (GameObject* flame = instance->GetGameObject(guid))
+                                flame->Respawn();
+                        }
+                    });
+                    });
```

Comment:
indent is wrong

Reply:
Same comment as above. I didn't write that.

Reply:
but you changed the indent

Reply:
You have changed the indentation of the entire file. So you fix it

Reply:
Yeah, and obviously I didn't realize that.